client/api_challenge.go

```diff
@@ -189,15 +245,45 @@ func (c *Client) AttestChallenge(ctx context.Context, submitterAddress, challeng
 	return resp.TxResponse, nil
 }
 
+// LatestAttestedChallenges will query the latest attested challenges (including heartbeat challenges).
```

Comment:
Add '-' between 
// LatestAttestedChallenges  and the brief introduction.

Reply:
Fixed